NEWS.txt

```diff
@@ -1,9 +1,9 @@
-PLEASE READ: MAXIMUM TTL EXPIRATION DATE NOTICE (CASSANDRA-14092)
-------------------------------------------------------------------
+PLEASE READ: MAXIMUM TTL EXPIRATION DATE NOTICE (CASSANDRA-14092 & CASSANDRA-14227)
+-----------------------------------------------------------------------------------
 (General upgrading instructions are available in the next section)
 
-The maximum expiration timestamp that can be represented by the storage engine is
-2038-01-19T03:14:06+00:00, which means that inserts with TTL thatl expire after
+The maximum expiration timestamp that can be represented by the storage engine has been
+raised to 2106-02-07T06:28:14+00:00, which means that inserts with TTL that expire after
```

Comment:
This would be the value if `Cell.MAX_DELETION_TIME` were `CassandraUInt.maxValueAsLong() - 1`, which is the maximum that we can represent while leaving `CassandraUInt.maxValueAsLong()` for `Cell.NO_DELETION_TIME`. 

However `Cell.MAX_DELETION_TIME` currently is `CassandraUInt.maxValueAsLong() - 2` for leaving space for some unknown future use.

If we are going to keep it that way, the formatted max timestamp mentioned here should be `2106-02-07T06:28:13+00:00`, one second earlier.

Reply:
I don't think we're totally aligned here.
- `-1` is to have room for the special value `LEGACY_OVERFLOWED_DELETION_TIME`
- `-2` is so we don't collide with the above one
- Yes I got 1s wrong good catch!

Reply:
Oh, right, makes sense, +1